### ISSUE TITLE
Unparsable image

### ISSUE THREAD

**Issue by rvangsgaard** [sksamuel/scrimage#63]
I have an image with the following attributes: PNG image data, 1440 x 2560, 8-bit/color RGB, non-interlaced. The image is attached.

I get this stack trace:
```
Caused by: java.lang.RuntimeException: Unparsable image
Nov 06 12:15:17 dg-admin-prod i-20aefec5: at com.sksamuel.scrimage.Image$$anonfun$apply$7.apply(Image.scala:790) ~[com.sksamuel.scrimage.scrimage-core_2.10-1.4.2.jar:1.4.2]
Nov 06 12:15:17 dg-admin-prod i-20aefec5: at com.sksamuel.scrimage.Image$$anonfun$apply$7.apply(Image.scala:790) ~[com.sksamuel.scrimage.scrimage-core_2.10-1.4.2.jar:1.4.2]
Nov 06 12:15:17 dg-admin-prod i-20aefec5: at scala.Option.getOrElse(Option.scala:120) [org.scala-lang.scala-library-2.10.4.jar:na]
Nov 06 12:15:17 dg-admin-prod i-20aefec5: at com.sksamuel.scrimage.Image$.apply(Image.scala:790) ~[com.sksamuel.scrimage.scrimage-core_2.10-1.4.2.jar:1.4.2]
Nov 06 12:15:17 dg-admin-prod i-20aefec5: at com.sksamuel.scrimage.Image$.apply(Image.scala:759) ~[com.sksamuel.scrimage.scrimage-core_2.10-1.4.2.jar:1.4.2] 
```
![screenshot_2014-11-03-13-47-23](https://cloud.githubusercontent.com/assets/144689/4935097/7d4b1ae2-65a8-11e4-9b1c-55e3111106fa.png)

**Comment by rvangsgaard** [comment]
This image is unparsable. Updating twelve-monkeys did not help.

This is not high priority, as I will switch to ImageMagick. Anyway, thanks for this library so far :)

![screenshot_2015-01-22-13-33-51](https://cloud.githubusercontent.com/assets/144689/5934900/6510ede4-a6d5-11e4-973c-0f4911bcc385.png)

**Comment by sksamuel** [comment]
This is fixed in 2.0 branch.
